frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -150,8 +155,8 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   if (!hasVMSettingsValueChanged(prevState, state, id, VMSettingsField.OPERATING_SYSTEM)) {
     return;
   }
-  const os = iGetVmSettingValue(state, id, VMSettingsField.OPERATING_SYSTEM);
-  const isWindows = os && os.startsWith('win');
+  const selectors = iGetRelevantTemplateSelectors(state, id);
```

Comment:
the prerequisites for this function got enhanced by this. The values will not update correctly, because they depend on the right template.

IMO it would be easier to move all the required logic to `mountBaseImageUpdater`